### PR TITLE
testing/iwd: new aport

### PR DESCRIPTION
**Pull request by J0WI**
Depends on #5790, some tests are failing.

### CODE REVIEW

File: testing/iwd/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,35 @@
+pkgname=iwd
```
Comment (andypost) [inline testing/iwd/APKBUILD]: no maintainer?

**Comment by andypost** [comment]
Also build fails
```
 build-aux/install-sh -c -d '/usr/bin'
 build-aux/install-sh -c -d '/usr/libexec'
  /bin/sh ./libtool   --mode=install /usr/bin/install -c client/iwctl monitor/iwmon '/usr/bin'
  /bin/sh ./libtool   --mode=install /usr/bin/install -c src/iwd '/usr/libexec'
libtool: install: /usr/bin/install -c client/iwctl /usr/bin/iwctl
install: can't create '/usr/bin/iwctl': Permission denied
make[2]: *** [Makefile:1350: install-binPROGRAMS] Error 1
make[2]: *** Waiting for unfinished jobs....
libtool: install: /usr/bin/install -c src/iwd /usr/libexec/iwd
install: can't create '/usr/libexec/iwd': Permission denied
make[2]: *** [Makefile:1399: install-libexecPROGRAMS] Error 1
```

**Comment by J0WI** [comment]
How can I fix the permission issues?

File: testing/iwd/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+check() {
+	cd "$builddir"
+	make check
+}
+
+package() {
+	cd "$builddir"
+	make install
```
Comment (bratkartoffel) [inline testing/iwd/APKBUILD]: I think this is the reason for the build error (permission denied).
I think this line should read like:
```make DESTDIR="$pkgdir" install```

Comment (bratkartoffel) [inline testing/iwd/APKBUILD]: I think this is the reason for the build error (permission denied).
I think this line should read like:
```make DESTDIR="$pkgdir" install```

Comment (J0WI) [reply testing/iwd/APKBUILD]: Thanks!

**Comment by bratkartoffel** [comment]
@J0WI Please add a comment on top of the APKBUILD about the maintainer / contributer.

**Comment by J0WI** [comment]
> Please add a comment on top of the APKBUILD about the maintainer / contributer.

I'm not sure about the responsibility of this module. Can I use the same as in `wpa_supplicant`?

**Comment by bratkartoffel** [comment]
You can just add your name there, at least for the contributor. If you're not comfortable with using your name / email address, feel free to use mine (e.g https://github.com/alpinelinux/aports/pull/5913/files)

**Comment by J0WI** [comment]
I've added the contributor line

**Comment by J0WI** [comment]
Superseded by ff2dbd7e2b9aeb75317fead3e1ddd086ecde71e9

**Pull request closed without merging**